Makefile.targets.mk

```diff
@@ -36,3 +36,22 @@ docker-build: build-all ## Build the envoy-gateway docker image.
 docker-push: ## Push the docker image for envoy-gateway.
 	@docker push $(IMAGE):$(TAG)
 
+.PHONY: lint
+lint: ## Run lint checks
+lint: lint-golint lint-yamllint lint-codespell
+
+.PHONY: lint-golint
+lint-golint:
```

Comment:
these targets should call the binary directly and not any intermediate wrapper script